tests/unit/crypto/test_bytearray.py

```diff
@@ -23,6 +23,7 @@ def test_operators(self):
         c = makeC()
         a &= c
         self.assertEqual(a, zero)
+        self.assertEqual(a & c, zero)
```

Comment:
I don't think this is right. `&=` should modify `a` "in-place", so it's just `a` we should be looking at.

Reply:
Oh, sorry, bad placement. I'm trying to cover the `__add__` method too, didn't mean to interfere with the `__iadd__` coverage.